definitions/npm/mersenne-twister_v1.x.x/flow_v0.33.x-/mersenne-twister_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'mersenne-twister' {
+  declare class $npm$MersenneTwister {
```

Comment:
You don't need to namespace your stuff inside the `declare module` body... it's already concealed and will not bleed to global scope